js/app/src/Blocks.svelte

```diff
@@ -502,7 +502,16 @@
 				});
 		} else {
 			if (dep.backend_fn) {
-				make_prediction();
+				if (dep.trigger_mode === "once") {
+					if (!dep.pending_request) make_prediction();
+				} else if (dep.trigger_mode === "multiple") {
+					do {
+						dep.pending_request = false;
+						make_prediction();
+					} while (dep.pending_request);
+				} else if (dep.trigger_mode === "always_last") {
+					make_prediction();
```

Comment:
I'm a little confused. Are "always_last" and "multiple" switched? Can you explain what's happening here: 

```js
					do {
						dep.pending_request = false;
						make_prediction();
					} while (dep.pending_request);
```